db/migrate/20211206191930_create_intakes.rb

```diff
@@ -0,0 +1,242 @@
+      t.datetime :sms_phone_number_verified_at
+      t.integer :sold_a_home, default: 0, null: false
+      t.integer :sold_assets, default: 0, null: false
+      t.string :source
+      t.integer :spouse_active_armed_forces, default: 0
+      t.string :spouse_auth_token
+      t.date :spouse_birth_date
+      t.integer :spouse_can_be_claimed_as_dependent, default: 0
+      t.integer :spouse_consented_to_service, default: 0, null: false
+      t.datetime :spouse_consented_to_service_at
+      t.inet :spouse_consented_to_service_ip
+      t.citext :spouse_email_address, index: { name: "index_intakes_on_spouse_email_address" }
+      t.integer :spouse_filed_2019, default: 0, null: false
+      t.string :spouse_first_name
+      t.integer :spouse_had_disability, default: 0, null: false
+      t.integer :spouse_issued_identity_pin, default: 0, null: false
+      t.string :spouse_last_name
+      t.string :spouse_middle_initial
+      t.integer :spouse_prior_year_agi_amount
+      t.string :spouse_prior_year_signature_pin
+      t.datetime :spouse_signature_pin_at
+      t.string :spouse_suffix
+      t.integer :spouse_tin_type
+      t.integer :spouse_was_blind, default: 0, null: false
+      t.integer :spouse_was_full_time_student, default: 0, null: false
+      t.integer :spouse_was_on_visa, default: 0, null: false
+      t.string :state
+      t.string :state_of_residence
+      t.string :street_address
+      t.string :street_address2
+      t.string :timezone
+      t.string :type, index: { name: "index_intakes_on_type" }
+      t.boolean :use_primary_name_for_name_control, default: false
+      t.boolean :viewed_at_capacity, default: false
+      t.string :visitor_id
+      t.references :vita_partner, index: true, foreign_key: true
+      t.string :vita_partner_name
+      t.integer :wants_to_itemize, default: 0, null: false
+      t.integer :was_blind, default: 0, null: false
+      t.integer :was_full_time_student, default: 0, null: false
+      t.integer :was_on_visa, default: 0, null: false
+      t.integer :widowed, default: 0, null: false
+      t.string :widowed_year
+      t.boolean :with_drivers_license_photo_id, default: false
+      t.boolean :with_general_navigator, default: false
+      t.boolean :with_incarcerated_navigator, default: false
+      t.boolean :with_itin_taxpayer_id, default: false
+      t.boolean :with_limited_english_navigator, default: false
+      t.boolean :with_other_state_photo_id, default: false
+      t.boolean :with_passport_photo_id, default: false
+      t.boolean :with_social_security_taxpayer_id, default: false
+      t.boolean :with_unhoused_navigator, default: false
+      t.boolean :with_vita_approved_photo_id, default: false
+      t.boolean :with_vita_approved_taxpayer_id, default: false
+      t.string :zip_code
+
+      t.timestamps
+    end
+
+    set_pk_sequence!(
+      'intakes',
+      '(SELECT MAX(id) FROM archived_intakes_2021)'
+    )
```

Comment:
tried googling this but probably still need an explanation

Reply:
want to make sure that the first id in the newly-created `intakes` table carries over from where `archived_intakes_2021` left off. prevents us from making mistakes where we confuse a 2022 intake id for a 2021 intake id